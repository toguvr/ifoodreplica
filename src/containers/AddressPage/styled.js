import styled from 'styled-components';
import TextField from '@material-ui/core/TextField';

export const ContainerAddressPage = styled.form `
    display: flex;
    align-items: center;
    flex-direction: column;
`

export const TextRegister = styled.div `
    font-size: 16px;
    color: #000000;
    height: 18px; 
    margin-top: 16px;  
`
export const InputAddress = styled(TextField) `
    margin-top: 20px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputNumber = styled(TextField) `
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputHouse = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputNeighborhood = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`
export const InputCity = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputState = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
    margin-bottom: 16px;
`

