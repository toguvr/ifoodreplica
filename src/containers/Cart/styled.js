import styled from 'styled-components';
import { Typography, FormControl } from '@material-ui/core';

export const CartWrapper = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
`;

export const DeliveryAdressContainer = styled.div`
  background-color: #eeeeee;
  width: 100%;
  height: 76px;
  padding: 25px;
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
`;

export const DeliveryAdress = styled.span`
  color: #b8b8b8;
`;

export const RestaurantAdress = styled.span`
  color: #b8b8b8;
`;

export const WaitingTime = styled.span`
  color: #b8b8b8;
`;

export const TotalText = styled.span`
  min-height: 21px;
  font-size: 18px;
  font-weight: bold;
  color: #e8222e;
`;

export const RestaurantDetailContainer = styled.div`
  width: 100%;
  height: 96px;
  padding: 25px;
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
`;

export const PriceContainer = styled.div`
  width: 100%;
  height: 76px;
  display: flex;
  flex-direction: column;
`;

export const FreightPrice = styled(Typography)`
  margin-top: 25px;
  padding-left: 70%;
`;

export const FinalPriceContainer = styled(Typography)`
  width: 100%;
  padding: 0 8%;
  display: flex;
  justify-content: space-between;
`;

export const PaymentContainer = styled.div`
  width: 100%;
  margin-top: 10px;
  display: flex;
  flex-direction: column;
  padding: 0 8%;
`;

export const ButtonDiv = styled.div`
  margin-bottom: 58px;
  width: 100%;
`;

export const HeaderDiv = styled.div`
  position: sticky;
  top: 0;
  width: 100%;
`;
export const EmptyCart = styled.p`
  opacity: 0.89;
  font-size: 16px;
  letter-spacing: -0.39px;
  color: #000000;
  margin-top: 20px;
`;

export const Error = styled.p`
  color: red;
`