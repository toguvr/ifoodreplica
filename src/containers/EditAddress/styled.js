import styled from 'styled-components';
import TextField from '@material-ui/core/TextField';

export const ContainerEditAddressPage = styled.div `
    display: flex;
    align-items: center;
    flex-direction: column;
`

export const InputAddress = styled(TextField) `
    margin-top: 24px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputNumber = styled(TextField) `
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputHouse = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputNeighborhood = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`
export const InputCity = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputState = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
    margin-bottom: 16px;
`

