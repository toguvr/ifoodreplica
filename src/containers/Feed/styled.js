import styled from 'styled-components'
import SearchIcon from '@material-ui/icons/Search';
import InputBase from '@material-ui/core/InputBase';

export const WrapperFixedComponents = styled.div`
  position: sticky;
  top: 0;
  background-color: #fff;
`
export const BaseInput = styled(InputBase) `
    margin-left: 17px;
`
export const IconSearch = styled(SearchIcon) `
    margin-left: 17px;
`
export const ContainerSearch = styled.div `
    width: calc(100% - 32px);
    height: 56px;
    border-radius: 2px;
    border: solid 1px #b8b8b8;
    display: flex;
    margin: 8px 16px;
    align-items: center;
` 