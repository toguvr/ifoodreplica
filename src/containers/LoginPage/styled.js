import styled from 'styled-components';
import TextField from '@material-ui/core/TextField';


export const ImgLogo = styled.img `
    width: 104px;
    height: 58px;
    margin-top: 88px;
`

export const ContainerLoginPage = styled.form `
    display: flex;
    align-items: center;
    flex-direction: column;
`

export const TextEnter = styled.div `
    font-size: 16px;
    color: #000000;
    height: 18px; 
    margin-top: 28px;  
`
export const InputEmail = styled(TextField) `
    margin-top: 20px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputPassword = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`

export const TextRegisterUser = styled.div `
    font-size: 16px;
    color: #000000;
    height: 18px; 
    margin-top: 28px;  
`

export const GoToSignUp = styled.a `
    color: black;
`

export const ErrorMessage = styled.p`
    color: red;
`