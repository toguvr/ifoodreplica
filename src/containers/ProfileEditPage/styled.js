import styled from 'styled-components';
import TextField from '@material-ui/core/TextField';

export const ContainerProfilePage = styled.div `
    display: flex;
    align-items: center;
    flex-direction: column;
`

export const InputName = styled(TextField) `
    margin-top: 24px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputEmail = styled(TextField) `
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`

export const InputCPF = styled(TextField)`
    margin-top: 16px;
    width: calc(100% - 32px);
    height: 56px;
`