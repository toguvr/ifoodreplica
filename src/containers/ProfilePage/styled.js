import styled from 'styled-components'

export const ProfileContainer = styled.div`
    width:calc(100% - 32px);
    margin:16px;
    display:flex;
    flex-direction:column; 
`

export const AdressContainer = styled.div`
    width:calc(100%);
    padding: 16px;
    display:flex;
    flex-direction:column; 
    background:#eeeeee; 
`

export const HistoryTitleContainer = styled.div`
    width:calc(100% - 32px);
    margin:16px 16px 0;
    display:flex;
    flex-direction:column; 
    border-bottom:1px solid black;
`

export const HistoryContainer = styled.div`
    width:calc(100% - 32px);
    margin:0 16px 58px;
    display:flex;
    flex-direction:column;
    align-items:center;
`

export const ProfileFlexDiv = styled.div`
    display:flex;
    justify-content:space-between;
`

export const ProfileText = styled.span`
    min-height: 18px;
    font-size: 16px;
    color: #000000;
    margin-bottom:8px;
`

export const AddressText = styled.span`
    min-height: 18px;
    font-size: 16px;
    color: #000000;
`

export const HistoryText = styled.span`
    min-height: 18px;
    font-size: 16px;
    color: #000000;
    margin-top:28px;
`

export const AdressTitle = styled.span`
    min-height: 18px;
    font-size: 16px;
    color: #000000;
    margin-bottom:8px;
    color: #b8b8b8;
`


