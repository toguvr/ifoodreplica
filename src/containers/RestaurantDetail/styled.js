import styled from 'styled-components';

export const Wrapper = styled.div``;
export const PrincipalDish = styled.p`
  font-size: 16px;
  letter-spacing: -0.39px;
  color: #000000;
  border-bottom: 1px solid black;
  padding-bottom: 8px;
  margin: 8px 16px 0;
`;
export const SideDish = styled.p`
  font-size: 16px;
  letter-spacing: -0.39px;
  color: #000000;
  border-bottom: 1px solid black;
  margin: 8px 16px;
  padding-bottom: 8px;
`;

export const CardDiv = styled.div`
  margin: 17px 16px 12px 16px;
  display: flex;
  flex-direction: column;
`;

export const CardImage = styled.img`
  height: 120px;
  object-fit: fill;
`;

export const CardDatesContainers = styled.div`
  display: flex;
  flex-direction: column;
  width: 100%;
  height: 126px;
  justify-content: space-around;
  margin-top: 12px;
`;

export const CardTitle = styled.p`
  color: #e8222e;
  font-size: 16px;
`;

export const CardDeliveryCategory = styled.p`
  color: #b8b8b8;
  font-size: 16px;
  margin: 0;
`;

export const CardDeliveryPriceRight = styled.span`
  color: #b8b8b8;
  font-size: 16px;
  min-height: 18px;
  margin: 0 0 0 18px;
`;

export const CardDeliveryTimeLeft = styled.span`
  color: #b8b8b8;
  font-size: 16px;
  min-height: 18px;
  margin: 0 18px 0 0;
`;

export const CardDeliveryAdress = styled.p`
  color: #b8b8b8;
  font-size: 16px;
  min-height: 18px;
`;

export const WrapperPrincipalDish = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;

  &:last-child{
    margin-bottom: 8px;
  }
`;

export const WrapperSideDish = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
`;
