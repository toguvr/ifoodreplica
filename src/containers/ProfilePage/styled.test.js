import React from "react";
import renderer from "react-test-renderer";
import { ProfileContainer, AdressContainer, HistoryTitleContainer, HistoryContainer, ProfileFlexDiv, ProfileText, AddressText, HistoryText, AdressTitle} from './styled'

describe("styleds",()=>{
    it("ProfileContainer",()=>{
        const tree = renderer.create(<ProfileContainer />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it("AdressContainer",()=>{
        const tree = renderer.create(<AdressContainer />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it("HistoryTitleContainer",()=>{
        const tree = renderer.create(<HistoryTitleContainer />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it("HistoryContainer",()=>{
        const tree = renderer.create(<HistoryContainer />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it("ProfileFlexDiv",()=>{
        const tree = renderer.create(<ProfileFlexDiv />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it("ProfileText",()=>{
        const tree = renderer.create(<ProfileText />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it("AddressText",()=>{
        const tree = renderer.create(<AddressText />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it("HistoryText",()=>{
        const tree = renderer.create(<HistoryText />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it("AdressTitle",()=>{
        const tree = renderer.create(<AdressTitle />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})