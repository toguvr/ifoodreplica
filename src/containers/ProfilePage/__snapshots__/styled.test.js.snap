// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styleds AddressText 1`] = `
<span
  className="sc-bZQynM hTJMiY"
/>
`;

exports[`styleds AdressContainer 1`] = `
<div
  className="sc-bwzfXH fiwDsq"
/>
`;

exports[`styleds AdressTitle 1`] = `
<span
  className="sc-htoDjs beNDqa"
/>
`;

exports[`styleds HistoryContainer 1`] = `
<div
  className="sc-bxivhb hXIiwd"
/>
`;

exports[`styleds HistoryText 1`] = `
<span
  className="sc-gzVnrw jKsnuh"
/>
`;

exports[`styleds HistoryTitleContainer 1`] = `
<div
  className="sc-htpNat dFXXDU"
/>
`;

exports[`styleds ProfileContainer 1`] = `
<div
  className="sc-bdVaJa hOPQam"
/>
`;

exports[`styleds ProfileFlexDiv 1`] = `
<div
  className="sc-ifAKCX hsHtRY"
/>
`;

exports[`styleds ProfileText 1`] = `
<span
  className="sc-EHOje cfYHCb"
/>
`;
